packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2605,7 +2605,6 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
       if (_tickersEnabled && _cursorActive) {
         _startCursorBlink();
       } else if (!_tickersEnabled && _cursorTimer != null) {
-        // Cannot use _stopCursorTimer because it would reset _cursorActive.
```

Comment:
Shouldn't this be changed to  -> `_stopCursorBlink?` IMHO the comment is useful as looking at the likes above it'd be tempting to do the opposite of `_startCursorBlink()`

```
  void _stopCursorBlink({ bool resetCharTicks = true }) {
    _cursorActive = false;
    _cursorBlinkOpacityController.value = 0.0;
    _cursorTimer?.cancel();
    _cursorTimer = null;
    if (resetCharTicks) {
      _obscureShowCharTicksPending = 0;
    }
  }
```

Reply:
I think you're right and the comment should remain, but should say _stopCursorBlink.